### ISSUE TITLE
What is the WER value of the LSR2 validation set？

### ISSUE THREAD

**Issue by qq2418916003** [ms-dot-k/Multi-head-Visual-Audio-Memory#4]
Thanks to the author for opening the source code, I kept dropping the WER of the validation set during the training, but the WER of the test set increased instead of decreasing

**Comment by ms-dot-k** [comment]
On the validation set of LRS2, we got about 42% WER and we watched a similar tendency with the test set WER.
Training on LRS2 and LRS3 datasets are difficult so diverse hyperparameters should be modified during training. These days, more improved training technologies for training LRS2 or LRS3 are released.

To achieve better performances, check the below lists.
[Augmentation]
1) Time masking
2) Spatial random erasing
3) Random horizontal flipping

[Curriculum Learning]
1) Use the number of frames instead of the number of words for curriculum learning [1]. 
Training with video lengths below 100, 150, 300, 450, and 600 for 50  epochs, respectively, gives more stable training and better results.

[1] https://arxiv.org/abs/2202.13084
